src/main/features/offer/views/your-offer.njk

```diff
@@ -0,0 +1,39 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, saveAndContinueButton, errorSummary, textArea %}
+{% from "date.njk" import dateInput %}
+{% set heading = t("Make an offer") %}
+{% set title = t('Money claim') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <div class="description-block">
+        <h4 class="heading-small"> {{ t('Your offer') }}</h4>
+      </div>
+
+      <form novalidate method="post">
+        {{ csrfProtection(csrf) }}
+
+        {{ textArea(label = t("You could offer a set amount, or to pay overdue rent, fix a problem, return goods."), 
```

Comment:
https://hmcts-cmc-prototype.herokuapp.com/dashboard/v2/offer/offer
shouldn't the hint be: 
> For example a payment, fix a problem or return goods.

and no label?

Reply:
looks like this is changed since I last looked at it.